### DISCUSSION TITLE
Action Detection With Pose Estimation

### DISCUSSION THREAD

**Discussion by Spawnfile** [ultralytics/ultralytics#5333] [General]
Hello everyone,

Which models are convenient for train a action detector with yolov8 pose estimation model to learn walking and standing etc. from keypoints ? 

Cheers

**Comment by glenn-jocher** [comment]
@Spawnfile hello!

For training an action detector that leverages pose estimation to recognize actions like walking and standing, you can use the YOLOv8 pose estimation model. This model is designed to detect keypoints, which can then be analyzed to infer different actions based on the spatial arrangement and movement of these points over time.

To train the model on such tasks, you would typically need a dataset labeled with keypoints corresponding to the human body and annotations that define the actions you want to detect (e.g., walking, standing). The model will learn to associate specific keypoint configurations and movements with the labeled actions.

Once you have your dataset ready, you can proceed with the training process using the Train mode and later validate your model with the Val mode. After training, you can use the Predict mode to test the model's ability to detect actions in new images or videos.

For more detailed guidance on each mode and task, please refer to the Ultralytics Docs, particularly the sections on Pose/Keypoint Estimation and the Train mode. Good luck with your project! 🚀

Cheers!
